sdk-java/src/test/java/ly/count/sdk/java/internal/ModuleDeviceIdTests.java

```diff
@@ -591,8 +591,12 @@ private void setupEvent() {
     private List<EventImpl> validateEvents(int requestIndex, String deviceId, int timedEventIdx) {
         List<EventImpl> existingEvents = TestUtils.readEventsFromRequest(requestIndex, deviceId);
         if (!existingEvents.isEmpty()) {
-            TestUtils.validateEvent(existingEvents.get(0), TestUtils.keysValues[2], null, 1, null, null); // casual event
-            TestUtils.validateEvent(existingEvents.get(timedEventIdx), TestUtils.keysValues[0], null, 1, null, 1.0); // timed event
+            String expectedCVId = "";
+            if (existingEvents.size() > 2) {
```

Comment:
I get expected current view id, from the 4th element. This is exact flow. and there is exactly 4 events at the queue. The last two ones are views so that is about it